packages/snaps-execution-environments/package.json

```diff
@@ -67,7 +68,7 @@
   "dependencies": {
     "@metamask/json-rpc-engine": "^10.0.2",
     "@metamask/object-multiplex": "^2.1.0",
-    "@metamask/post-message-stream": "^9.0.0",
+    "@metamask/post-message-stream": "^10.0.0",
```

Comment:
`@metamask/post-message-stream` only had a CJS version, and since it's using `@metamask/utils`, it caused Webpack to use the CJS version of `@metamask/utils` while an ESM version was available. After bumping this to `10.0.0`, we can use the ESM version of both `@metamask/post-message-stream` and `@metamask/utils`, resulting in a smaller bundle.